### PR TITLE
FloatingButtons: Add environment badge to floating buttons container

### PR DESCRIPTION
**Pull request by spen**
> Note this PR is branching off of #26955. Once that's landed, we'll base this PR off of master.

As a follow up to #26955, this PR adds the dev `EnvironmentBadge` component to the floating buttons container. Until now, it's been positioned as a `fixed` element with `right` and `bottom` attributes marking it's exact position. Though this works, it's not adaptive to changes made to other near-by elements and instead would require manual fiddles to be made as we make changes to things like `InlineHelp`.

### To Test

- Patch this PR and go to http://calypso.localhost:3000/
- You should notice the dev bar in the bottom right
  - Does it look correct? Does the animation (on hover) still work as expected?
- Try increasing the size of inline-help, you should see the env badge positioning adjust naturally:
```js
document.querySelector( '.inline-help' ).style.width = '200px';
```

### CODE REVIEW

File: client/layout/floating-actions/index.jsx [hunk 1]
```diff
@@ -28,7 +29,7 @@ const FloatingActions = ( { isHappychatButtonVisible } ) => (
 			{ isHappychatButtonVisible &&
 				config.isEnabled( 'happychat' ) && <HappychatButton allowMobileRedirect /> }
 		</div>
-		{ /* TODO: Move environment badge here */ }
+		<EnvironmentBadge />
```
Comment (andfinally) [inline client/layout/floating-actions/index.jsx]: Now I see why there's a separate div for the vertical items!

File: client/document/desktop.jsx [hunk 2]
```diff
@@ -116,6 +80,20 @@ class Desktop extends React.Component {
 							} }
 						/>
 					) }
+					{ env === 'development' && (
+						<script
+							type="text/javascript"
```
Comment (andfinally) [inline client/document/desktop.jsx]: Can we do without the `type` attribute?

File: client/layout/floating-actions/index.jsx (hunk 1 above)
Comment (spen) [reply client/layout/floating-actions/index.jsx]: :P Yeah sorry I didn't get around to explaining that yet! I planned on opening this then linking it through!
I did have separate wrapper components originally, handling verticals and horizontals but it turns out that only the vertical group was needed :)
If we want to position something above `<EnvironmentBadge />`, it should be as simple as wrapping the two in that class 👌

**Comment by mattsherman** [comment]
@spen What's the status of this PR? Is it still actively being worked on, or should it be closed?

**Comment by mattsherman** [comment]
@spen Okay. I'll move them to the HACK Week column of the Samus Backlog so that we don't forget about them!

**Comment by spen** [comment]
Thanks Meeps!

**Comment by sirreal** [comment]
Closing as abandoned.

**Pull request closed without merging**